### PR TITLE
Aleksei/graphql parameter change

### LINKED ISSUE #3415: Add parameter to persist API change.
Add parameter that will persist API url in localStorage and change current `graphql` parameter to `api`.

### PR DESCRIPTION
**Pull request by iambeone**
Closes #3415 

Changes `graphql` parameter to `api`.
Adds `iwouldliketochangetheapipersistentlyandiknowwhatido` paremeter that persistently changes API url and store its value in localStorage

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/gql/apollo.js [hunk 1]
```diff
@@ -12,8 +12,47 @@ import config from "src/../config"
 
 Vue.use(VueApollo)
 
+// persistent api
+// setting api url in localStorage
+const checkPersistentAPI = urlParams => {
```
Comment (faboweb) [inline src/gql/apollo.js]: needs to be added to ActionManager as well

File: src/gql/apollo.js [hunk 2]
```diff
@@ -12,8 +12,47 @@ import config from "src/../config"
...
+  if (
+    typeof urlParams.iwouldliketochangetheapipersistentlyandiknowwhatido !==
```
Comment (jbibla) [inline src/gql/apollo.js]: LOL

File: src/ActionModal/utils/ActionManager.js [hunk 3]
```diff
@@ -77,16 +77,11 @@ export default class ActionManager {
...
-      (urlParams.graphql ? decodeURIComponent(urlParams.graphql) : false) ||
-      config.graphqlHost
-    const urlParams = getURLParams(window)
+    const graphqlHost = getGraphqlHost()
 
-    return fetch(
-      `${graphqlHost(urlParams)}/transaction/${command}`,
-      options
-    ).then(r => r.json())
+    return fetch(`${graphqlHost}/transaction/${command}`, options).then(r =>
```
Comment (jbibla) [inline src/ActionModal/utils/ActionManager.js]: what is `r`? can you use the whole word?

Comment (faboweb) [reply src/ActionModal/utils/ActionManager.js]: @iambeone you missed this comment. Do you check your GitHub notifications?

Comment (iambeone) [reply src/ActionModal/utils/ActionManager.js]: Yes, I saw it, but forgot to react. I will be more careful

Comment (faboweb) [reply src/ActionModal/utils/ActionManager.js]: changed it

**Pull request merged**